.gitignore

```diff
@@ -41,3 +41,7 @@ hs_err_pid*
 # MacOS Garbage
 .DS_Store
 /*/.DS_Store
+
+.classpath
+.project
+*/.settings
```

Comment:
If you are going to ignore these files, then you should likely delete the existing files from github.

Reply:
Good idea -> https://github.com/OpenLiberty/open-liberty-misc/pull/13